packages/material-ui/src/Unstable_TrapFocus/Unstable_TrapFocus.d.ts

```diff
@@ -18,7 +18,7 @@ export interface TrapFocusProps {
   /**
    * A single child content element.
    */
-  children: React.ReactNode;
+  children?: React.ReactNode;
```

Comment:
You can remove the `typescript-to-proptypes-ignore` directives for these cases so that the runtime matches the types.

Reply:
I don't understand. The above change doesn't change the generated PropType.

Reply:
Oh these are the nullable nodes. I thought you changed the non-nullable ones. Not sure why we value consistency over semantics but I don't want to argue over non-essential changes.

Reply:
What about we go in the other direction in order to avoid potential confusion?

```jsx
children?: React.ReactNode;
```